test/with_dummyserver/test_https.py

```diff
@@ -529,6 +529,11 @@ class TestHTTPS_IPSAN(HTTPSDummyServerTestCase):
 
     def test_can_validate_ip_san(self):
         """Ensure that urllib3 can validate SANs with IP addresses in them."""
+        try:
+            import ipaddress  # noqa
```

Comment:
If you're using Flake8 3.x, you can use `# noqa: F...` to specify just that code.